securetea/core.py

```diff
@@ -93,6 +133,9 @@ def on_move(self, x, y):
         # Send a warning message via twitter account
         self.twitter.notify(msg)
 
+        # Send a warning message via telegram bot
+        self.telegram.notify(msg)
```

Comment:
@mishal23 We are not sure, user has provided Telegram details or not, if not program will break.

Reply:
Right, my bad. Will update.